jib-core/src/main/java/com/google/cloud/tools/jib/configuration/BuildConfiguration.java

```diff
@@ -228,22 +137,22 @@ public BuildConfiguration build() {
           }
           return new BuildConfiguration(
               buildLogger,
-              creationTime,
               baseImageReference,
-              baseImageCredentialHelperName,
-              knownBaseRegistryCredentials,
+              baseImageConfiguration.getCredentialHelper(),
```

Comment:
These should just use the configuration objects directly, since they are immutable.